### ISSUE TITLE
Tricolore error message

### ISSUE THREAD

**Issue by elj970** [jschoeley/tricolore#1]
Hi,
I'm getting an error message when I try and use the Tricolore function. My data and code work on a friend's computer and I've tried changing my version of R to the one that they have and have also tried updating to the most recent version of R but I get the error message in both cases. My supervisor has tried to run it on his computer and also gets the error. I've looked online but I haven't been able to find any indication of what the problem is.

I have attached screenshots of the session info for the computer it works on and the session info for my computer, which includes my code and the error message.  My data frame contains 3 columns, with each row adding up to 100.

![screen shot_with_error](https://user-images.githubusercontent.com/31698935/42934886-334347f0-8b40-11e8-8882-22dee1253c6e.png)

![screen shot_works](https://user-images.githubusercontent.com/31698935/42934525-594bd922-8b3f-11e8-9a18-c130807b137a.png)

Cheers,
Emma

**Comment by jschoeley** [comment]
Thank you Emma.

The recent update of ggplot2 to version 3.0.0 caused ggtern to break and in turn tricolore. We're working on a fix.

In the meantime, in order to use tricolore you must revert to ggplot version 2.2.1.

```r
install.packages("devtools")
devtools::install_version("ggplot2", version = "2.2.1")
devtools::install_version("ggtern", version = "2.2.1")
```

Sorry for the inconvenience.

Best,

Jonas

**Comment by elj970** [comment]
Thank you! I've got it working now.

**Comment by antuki** [comment]
I just had the same issue. Can't wait to enjoy using this nice package with sf and geom_sf from new ggplot2  ! Can you write a message here when it is corrected  ? Thank you  !

**Comment by jschoeley** [comment]
Sure thing, I will close this issue once ggtern, and in turn tricolore, work with the newest ggplot version.

**Comment by jschoeley** [comment]
Fixed by b3a09f3e1428ceee2f4b624c84a4605c4be08024

Please run `install.packages("ggtern")` to update to the latest version of ggtern.
